drivers/bt_ll_nrfxlib/clock_control/clock_control_ll_nrfxlib.c

```diff
@@ -16,6 +17,13 @@
 #include <hal/nrf_power.h>
 #endif
 
+static bool is_context_atomic(void)
+{
+	int key = irq_lock();
+	bool is_atomic = !z_arch_irq_unlocked(key) || z_is_in_isr();
+	irq_unlock(key);
+}
```

Comment:
Shouldn't `is_atomic` be returned here?

Reply:
My bad. I'm surprised that CI tests did not catch this